app/volume.go

```diff
@@ -293,7 +325,7 @@ func (job *Job) snapshotAndCleanup() (err error) {
 
 	defer func() {
 		if err != nil {
-			logrus.Warnf("created snapshot successfully but errored on cleanup for %v: %v", volumeName, err)
+			job.logger.Warnf("created snapshot successfully but errored on cleanup for %v: %v", volumeName, err)
```

Comment:
consider using logger.withError for the error.

Reply:
Thank you. Fixed